### PR TITLE
Bio1 2022 q2 game of drones

### PR DESCRIPTION
**Pull request by AG-88301**

### CODE REVIEW

**Comment by AG-88301** [comment]
I think this might be one of my better editorials... time to get ykr to prove me wrong

**Review by ykr5184 — CHANGES_REQUESTED** [review]
This is really good, and very nicely laid out, especially for a problem which could be a real mess if written poorly. Your code is also beautiful. I have just suggested a few slight edits below.

File: solutions/bio1/GameOfDrones/index.tsx [hunk 1]
```diff
@@ -19,11 +19,13 @@ export const GameOfDrones = () => {
       <SSubtitle>Skirmish: Claiming an Edge</SSubtitle>
       <SText>
         To claim an edge, we can firstly update the state of the edge in the <SCode>Hexagon</SCode> class.
-        However, we must also consider that updating a shared edge also will update the state of it in the adjacent hexagon.
+        However,  must be aware that updating a shared edge will also update the state of it in the adjacent hexagon.
```
Comment (ykr5184) [inline solutions/bio1/GameOfDrones/index.tsx:22]: Add "we".

**Review by ykr5184 — DISMISSED** [review]
Great!

**Review by ykr5184 — APPROVED** [review]
Sorry about the failed deployment. Should be fixed now.

**Comment by ykr5184** [comment]
Hi, can someone else add a review, because it won't let me approve it because I messed up the merge and had to push a commit to fix it

**Review by macaquedev — APPROVED** [review]

**Pull request merged**